static/js/components/TokenList.jsx

```diff
@@ -26,67 +35,53 @@ const TokenList = ({ tokens }) => {
       <h3>
         My Tokens
       </h3>
-      <table>
-        <tbody>
-          <tr>
-            <td>
-              <b>
-                Value
-              </b>
-              &nbsp;&nbsp;
-            </td>
-            <td>
-              <b>
-                Name
-              </b>
-              &nbsp;&nbsp;
-            </td>
-            <td>
-              <b>
-                ACLS
-              </b>
-              &nbsp;&nbsp;
-            </td>
-            <td>
-              <b>
-                Created
-              </b>
-              &nbsp;&nbsp;
-            </td>
-            <td>
-              <b>
-                Delete
-              </b>
-            </td>
-          </tr>
+      <Table size="small" component={Paper}>
```

Comment:
Any way to make this part work without getting these browser warnings? `Warning: validateDOMNesting(...): <tbody> cannot appear as a child of <div>.`

Reply:
How about wrapping `Table` in `Paper` instead, as in this example? https://codesandbox.io/s/2omrn3oq30?file=/demo.js

Reply:
Can confirm wrapping Table in Paper instead does not produce any warnings.

Reply:
Fixed, thanks!